File: src/program/go/fizzbuzz/fizzbuzz.go
package main

import (
	"fmt"
	"strconv"
)

func fizzbuzzString(n int) string {
	n3 := n%3 == 0
	n5 := n%5 == 0
	if n3 && n5 {
		return "fizz buzz"
	} else if n3 {
		return "fizz"
	} else if n5 {
		return "buzz"
	} else {
		return strconv.Itoa(n)
	}
}

func printFizzbuzz(n int, m int) {
	for k := n; k <= m; k++ {
		fmt.Println(fizzbuzzString(k))
	}
}

func main() {
	printFizzbuzz(1, 30)
}

File: src/program/nim/fizzbuzz/src/fizzbuzz.nim
for n in 1..50:
  if n mod 3 == 0 and n mod 5 == 0:
    echo "fizzbuzz"
  elif n mod 3 == 0:
    echo "fizz"
  elif n mod 5 == 0:
    echo "buzz"
  else:
    echo n